release-tools/scripts/setup_runners_service.sh

```diff
@@ -18,7 +19,7 @@
 ###############################################################################################
 
 # This script allows users to manually assign parameters
-if [ "$#" -ne 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
+if [ "$#" -lt 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
 then
   echo "Please assign 2 parameters when running this script"
   echo "Example: $0 \$SETUP_DISTRO \$SETUP_ACTION"
```

Comment:
Please also update the help message with $ARCHITECTURE

Reply:
Added a message that the ARCHITECTURE is optional.

Reply:
Where is the message? not seeing it.

Reply:
I added a comment where the change is made. Please reply.